### PR TITLE
Checkstyle errors fixed - Button.java + DefeatScreen.java + PausedScreen.java + VictoryScreen.java

### PR DESCRIPTION
**Pull request by peace-4-you**
All checkstyle errors fixed but one. TODO tag error is an expected error

Update: includes checkstyle fixes to DefeatScreen, PausedScreen and VictoryScreen as well.

### CODE REVIEW

File: bust-a-move-framework/src/main/java/nl/tudelft/bust_a_move20162017/bust_a_move_framework/gamestate/Button.java [hunk 1]
```diff
@@ -16,31 +9,54 @@
...
+     * Font size constant.
+     */
+    private final int fontSize = 20;
+    /**
+     * Font of the text.
+     */
     //TODO: Find a nice font.
-    public TrueTypeFont font = new TrueTypeFont(new Font("Verdana", Font.BOLD, 20), true);
+    private TrueTypeFont font = new TrueTypeFont(
+            new Font("Verdana", Font.BOLD, fontSize), true);
```
Comment (mauricewillemsen) [inline bust-a-move-framework/src/main/java/nl/tudelft/bust_a_move20162017/bust_a_move_framework/gamestate/Button.java]: Fontsize can be refered to GameConfig.TEXT_SIZE

File: bust-a-move-framework/src/main/java/nl/tudelft/bust_a_move20162017/bust_a_move_framework/gamestate/Button.java [hunk 2]
```diff
@@ -16,31 +9,54 @@
...
+     */
     private float bHeight;
+    /**
+     * Font size constant.
+     */
+    private final int fontSize = 20;
+    /**
+     * Font of the text.
+     */
     //TODO: Find a nice font.
```
Comment (mauricewillemsen) [inline bust-a-move-framework/src/main/java/nl/tudelft/bust_a_move20162017/bust_a_move_framework/gamestate/Button.java]: Remove the todo, font is fine

**Comment by mauricewillemsen** [comment]
You can refer the font size to the new GameConfig class and remove the getter. Font size won't be used outside anymore. So the commits to three 3 screen classes aren't necesary, those will be fixed in #46

**Comment by peace-4-you** [comment]
If we can merge your branch first, then I'll make the fontSize changes. It avoids merge conflict later.
Also, if I change my code now, the program will not build.

**Pull request merged**